doc/man/pam_cockpit_cert.xml

```diff
@@ -0,0 +1,168 @@
+<refentry id="pam_cockpit_cert.8">
+
+  <!--
+  This file is part of Cockpit.
+
+  Copyright (C) 2019 Red Hat, Inc.
+
+  Cockpit is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+
+  Cockpit is distributed in the hope that it will be useful, but
+  WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public License
+  along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+  <refentryinfo>
+    <title>pam_cockpit_cert</title>
```

Comment:
This whole binary seems like an implementation detail. Are we sure we want to document it?

Reply:
Maybe move this to doc/ folder as a markdown file instead?

Reply:
As it appears in /etc/pam.d/, I felt that some admins would like to know what it is and how to configure it. So in that sense, that's the only API that we have with this PR right now.